src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -871,29 +932,37 @@
       }
     }
 
-    .pagination-current {
-      font-weight: 600;
-      border-bottom: (2 * $clr-default-borderwidth) solid $action-blues-dark-midtone;
-    }
-
-    $clr-pagination-extra-margin: 0.25rem;
+    $clr-pagination-extra-margin: 0.5rem;
+    .pagination-first,
+    .pagination-last,
     .pagination-previous,
     .pagination-next {
-      display: inline-block;
-      vertical-align: middle;
+      display: flex;
+      align-items: center;
       width: $clr-datagrid-icon-size;
       height: $clr-datagrid-icon-size;
       background-repeat: no-repeat;
       background-size: contain;
     }
 
+    .pagination-first:disabled,
+    .pagination-last:disabled,
+    .pagination-previous:disabled,
+    .pagination-next:disabled {
+      clr-icon {
```

Comment:
generally, we try not to target `clr-icon` elements directly. Also we have a number of hardcoded color values in here we should clean up

Reply:
What is the general approach we recommend in this case? Use classnames on these icons?